src/shared-components/Toolbar/Toolbar.module.css

```
@@ -0,0 +1,103 @@
+  white-space: nowrap;
+}
+
+/* Screen reader only */
+.srOnly {
+  position: absolute;
+  width: var(--border-1);
+  height: var(--border-1);
+  padding: 0;
+  margin: calc(-1 * var(--border-1));
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)